src/inspector_agent.cc

```diff
@@ -536,12 +448,17 @@ void AgentImpl::Stop() {
   delete inspector_;
 }
 
+bool AgentImpl::IsConnected() {
+  return delegate_ != nullptr && delegate_->IsConnected();
+}
+
 bool AgentImpl::IsStarted() {
   return !!platform_;
 }
 
 void AgentImpl::WaitForDisconnect() {
   shutting_down_ = true;
+  Write(0, StringView());
```

Comment:
This is a temporary "command" to stop accepting new connections. In next pull request there will be a better API with meaningful requests as opposed to magical strings...